include/cantera/thermo/BinarySolutionTabulatedThermo.h

```diff
@@ -147,12 +147,19 @@ class BinarySolutionTabulatedThermo : public IdealSolidSolnPhase
     virtual void getParameters(AnyMap& phaseNode) const;
     virtual void initThermoXML(XML_Node& phaseNode, const std::string& id_);
 
+    void getIntegratedPartialMolarVolumes(doublereal* vbar);
+    void getPartialMolarVolumes(doublereal* vbar) const;
```

Comment:
Please avoid using `doublereal` in new code, preferring the standard `double` instead.